sentry/resource_sentry_key.go

```diff
@@ -97,7 +99,10 @@ func resourceSentryKeyRead(d *schema.ResourceData, meta interface{}) error {
 	project := d.Get("project").(string)
 
 	keys, _, err := client.ProjectKeys.List(org, project)
-	if err != nil {
+	if err != nil && err.Error() == errProjectNotFound {
```

Comment:
Would it be better to test the response status code instead?

```suggestion
	keys, resp, err := client.ProjectKeys.List(org, project)
	if err != nil {
		if resp.StatusCode == http.StatusNotFound {
			d.SetId("")
			return nil
		}
		return err
	}
```

Reply:
Yeah, true, sorry I didn't look at the underlying API. Since the underlying library seems to be yours, too, perhaps you could consider making the HTTP response code part of the error itself?